datafusion/physical-plan/src/spill/spill_manager.rs

```diff
@@ -126,14 +125,11 @@ impl SpillManager {
         &self,
         spill_file_path: RefCountedTempFile,
     ) -> Result<SendableRecordBatchStream> {
-        let mut builder = RecordBatchReceiverStream::builder(
+        let stream = Box::pin(SpillReaderStream::new(
             Arc::clone(&self.schema),
-            self.batch_read_buffer_capacity,
-        );
-        let sender = builder.tx();
+            spill_file_path,
+        ));
 
-        builder.spawn_blocking(move || read_spill(sender, spill_file_path.path()));
```

Comment:
Is read_spill used anywhere else? It seems like this PR's approach is strictly better and so we could entirely remove the old method

Reply:
it's already deleted :)

Reply:
🤦